ofdb-core/src/usecases/send_update_reminders.rs

```
@@ -0,0 +1,267 @@
+    Ok(places)
+}
+
+fn find_unsent_reminders<R>(
+    repo: &R,
+    outdated_places: Vec<Place>,
+    target_contact: TargetContact,
+    reminder_interval: Duration,
+) -> Result<Vec<Reminder>>
+where
+    R: SubscriptionRepo + ReminderRepo + UserRepo,
+{
+    let mut reminders = vec![];
+    match target_contact {
+        TargetContact::Owner => {
+            for p in outdated_places {
+                if let Some(email) = p.contact_email() {
+                    if send_new_reminder(repo, &p.id, email, reminder_interval) {
+                        reminders.push(Reminder {
+                            recipients: vec![email.clone()],
+                            last_change: p.created.at,
+                            place: p,
+                        });
+                    }
+                }
+            }
+        }
+        TargetContact::Scout => {
+            let subscriptions = repo.all_bbox_subscriptions()?;
+            let users = repo.all_users()?;
+            for p in outdated_places {
+                let scouts = get_scouts_subscribed_to_place(&p, &users, &subscriptions);
+                let mut recipients = vec![];
+                for s in scouts {
+                    let email = EmailAddress::from(s.email.clone());
+                    if send_new_reminder(repo, &p.id, &email, reminder_interval) {
+                        recipients.push(email);
+                    }
+                }
+                if !recipients.is_empty() {
+                    reminders.push(Reminder {
+                        recipients,
+                        last_change: p.created.at,
+                        place: p,
+                    });
+                }
+            }
+        }
+    }
+    Ok(reminders)
+}
+
+fn send_new_reminder<R>(
+    repo: &R,
+    place_id: &Id,
+    email: &EmailAddress,
+    reminder_interval: Duration,
+) -> bool
+where
+    R: ReminderRepo,
+{
+    match repo.find_last_sent_reminder(place_id, email).unwrap() {
```

`unwrap()` -> `ok()` if you really want to silently ignore any errors. Errors should at least be logged as warnings.